src/bin/smooth_scale.rs

```diff
@@ -130,7 +129,7 @@ fn smooth<const D: usize>(_self: &Rule<D>, rule_idx: usize, orig: &Image, buffer
                     buffer.plot_line_width(a, b, 1.);
                     buffer.plot_line_width(b, c, 1.);
                     buffer.plot_line_width(c, a, 1.);
-                    let centroid = ((a.0 + b.0 + c.0) / 3, (a.1 + b.1 + c.1) / 3);
+                    let _centroid = ((a.0 + b.0 + c.0) / 3, (a.1 + b.1 + c.1) / 3);
```

Comment:
Better delete this, this was used to fill the triangle with regular flood fill but at the end I used `fill_triangle()`

Reply:
done